network/game.go

```diff
@@ -6,41 +6,94 @@ import (
 	"github.com/joshprzybyszewski/cribbage/model"
 )
 
+type CreateGameRequest struct {
+	PlayerIDs []model.PlayerID `json:"playerIDs"`
+}
+
+type CreateGameResponse struct {
+	ID              model.GameID              `json:"id"`
+	Players         []Player                  `json:"players"`
+	PlayerColors    map[model.PlayerID]string `json:"player_colors,omitempty"`
+	BlockingPlayers map[model.PlayerID]string `json:"blocking_players,omitempty"`
+	CurrentDealer   model.PlayerID            `json:"current_dealer"`
+}
+
+func ConvertToCreateGameResponse(g model.Game) CreateGameResponse {
+	return CreateGameResponse{
+		ID:              g.ID,
+		Players:         convertToPlayers(g.Players),
+		PlayerColors:    convertToColors(g.PlayerColors),
+		BlockingPlayers: convertToBlockingPlayers(g.BlockingPlayers),
+		CurrentDealer:   g.CurrentDealer,
+	}
+}
+
+type GetGameResponse struct {
+	ID              model.GameID              `json:"id"`
+	Players         []Player                  `json:"players"`
+	PlayerColors    map[model.PlayerID]string `json:"player_colors,omitempty"`
+	CurrentScores   map[string]int            `json:"current_scores"`
+	LagScores       map[string]int            `json:"lag_scores"`
+	Phase           string                    `json:"phase"`
+	BlockingPlayers map[model.PlayerID]string `json:"blocking_players,omitempty"`
+	CurrentDealer   model.PlayerID            `json:"current_dealer"`
+	Hands           map[model.PlayerID][]Card `json:"hands,omitempty"`
+	Crib            []Card                    `json:"crib,omitempty"`
+	CutCard         Card                      `json:"cut_card"`
+	PeggedCards     []PeggedCard              `json:"pegged_cards,omitempty"`
+}
+
 func ConvertToGetGameResponse(g model.Game) GetGameResponse {
-	currentScores, lagScores := convertScores(g.CurrentScores, g.LagScores)
+	currentScores, lagScores := convertToScores(g.CurrentScores, g.LagScores)
 	return GetGameResponse{
 		ID:              g.ID,
-		Players:         newPlayersFromModels(g.Players),
-		PlayerColors:    convertColors(g.PlayerColors),
+		Players:         convertToPlayers(g.Players),
+		PlayerColors:    convertToColors(g.PlayerColors),
 		CurrentScores:   currentScores,
 		LagScores:       lagScores,
-		Phase:           g.Phase.String(),
-		BlockingPlayers: convertBlockingPlayers(g.BlockingPlayers),
+		Phase:           convertToPhase(g.Phase),
+		BlockingPlayers: convertToBlockingPlayers(g.BlockingPlayers),
 		CurrentDealer:   g.CurrentDealer,
-		CutCard:         newCardFromModel(g.CutCard),
-		PeggedCards:     convertPeggedCards(g.PeggedCards),
+		CutCard:         convertToCard(g.CutCard),
+		PeggedCards:     convertToPeggedCards(g.PeggedCards),
 	}
 }
 
 func ConvertToGetGameResponseForPlayer(g model.Game, pID model.PlayerID) (GetGameResponse, error) {
-	pIsInGame := false
+	playing := false
```

Comment:
:cactus: I wasn't super happy with `pIsInGame`, but I feel like we lose a bit of clarity here? `playing` doesn't _necessarily_ imply that we're talking about _this_ game

Reply:
how about `isPlaying`?